default/stringtables/en.txt

```diff
@@ -8261,6 +8270,10 @@ Building
 OBJ_SHIP
 Ship
 
+# As the OBJ_SHIP and OBJ_BUILDING are used in the production window, we use OBJ_PROJECT for BT_STOCKPILING
+OBJ_PROJECT
```

Comment:
a project is not an object, don't suggest it is in this way

Reply:
ok